### ISSUE TITLE
can't install on ruby 2.2.1 on windows with bundler

### ISSUE THREAD

**Issue by net1957** [kubo/ruby-oci8#73]
Hello!

I use ruby-oci8 on windows 8.1 x64:
````
ruby 2.1.5p273 (2014-11-13 revision 48405) [x64-mingw32]
ruby 2.2.1p85 (2015-02-26 revision 49769) [x64-mingw32]
````
for ruby 2.1.5, 'gem install ruby-oci8' install the precompiled one an all is working.
for ruby 2.2.1 the same precompiled gem is installed , but it wont work (binary compatibilty between rubies)
for all 2 rubies if I install with gem install ruby-oci8 --platform=ruby all is working well 

But I use bundler and i can't find a way to make it work on ruby 2.2.1. Bundler always get the precompiled one from rubygems.

I did open a issue on bundler see https://github.com/bundler/bundler/issues/3488

Any idea to make it work ?

versions:
Windows 8.1 x64
ruby 2.1.5p273 (2014-11-13 revision 48405) [x64-mingw32]
ruby 2.2.1p85 (2015-02-26 revision 49769) [x64-mingw32]
bundler (1.8.5)
rubygem 2.4.6
ruby-oci8 (2.1.7 ruby x64-mingw32)

**Comment by kubo** [comment]
I'll release ruby-oci8 2.1.8, which bundles precompile binary for ruby 2.2.1, next weekend.

**Comment by kubo** [comment]
It needs more time to release ruby-oci8 2.1.8. One of tests got access violation with ruby 2.2.1. I'll relase ruby-oci8 2.1.8 after the issue is fixed.

**Comment by net1957** [comment]
I can wait a little as it work on ruby 2.1.5. Nice to see your work on it. Regards

**Comment by kubo** [comment]
I released ruby-oci8 2.1.8. It works with ruby 1.8.6 - 2.1.1 (Windows 32bit) and ruby 2.0.0 - 2.2.1 (Windows x64).

**Comment by net1957** [comment]
Nice job! Thanks
